test/e2e_node/critical_pod_test.go

```diff
@@ -122,7 +131,7 @@ func getNodeCPUAndMemoryCapacity(f *framework.Framework) v1.ResourceList {
 	}
 }
 
-func getTestPod(critical bool, name string, resources v1.ResourceRequirements) *v1.Pod {
+func getTestPod(critical bool, name string, resources v1.ResourceRequirements, scp *schedulingv1.PriorityClass) *v1.Pod {
```

Comment:
I don't think we need to pass this priorityclass in.  Just set it in the pod based on the `critical` bool.

Reply:
+1 if we can name `pod.Spec.PriorityClassName` without `f := framework.NewDefaultFramework("critical-pod-test")` which is defined outside of this function.